### PR TITLE
feat(ProfilePopup): Add avatar change functionality

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #3734

### CODE REVIEW

**Review by PascalPrecht — APPROVED** [review]
LGTM! Well done!

**Review by alexandraB99 — APPROVED** [review]

**Comment by anastasiyaig** [comment]
@borismelnik would u mind to show  a picture preview instead of empty circle?

this is how it looks when editing profile from online/offline identifier (new functionality u added)

Screenshot 2021-12-23 at 13 48 07

and this is how it looks like when editing profile from Settings

Screenshot 2021-12-23 at 13 48 33

**Comment by borismelnik** [comment]
@alexandraB99 It will be great if you find time for checking proper use of the store.

File: ui/app/AppMain.qml [hunk 1]
```diff
@@ -15,6 +15,7 @@ import "./AppLayouts/Chat/popups"
 import "./AppLayouts/Chat/popups/community"
 import "./AppLayouts/Profile/Sections"
 import "./AppLayouts/stores"
+import "./AppLayouts/Profile/stores" as ProfileStores
```
Comment (alexandraB99) [inline ui/app/AppMain.qml:18]: There is no need for an additional store instance, you can add this in the `status-desktop/ui/app/AppLayouts/stores/RootStore.qml` and assign it in line 462 like`store: appMain.rootStore.profileStore`.

**Review by alexandraB99 — CHANGES_REQUESTED** [review]

**Review by Khushboo-dev-cpp — APPROVED** [review]

**Review by noeliaSD — APPROVED** [review]

**Review by alexandraB99 — APPROVED** [review]

**Pull request merged**